spark-jobs/src/main/scala/za/co/absa/enceladus/common/plugin/enceladus/EnceladusRunUrl.scala

```diff
@@ -13,12 +13,12 @@
  * limitations under the License.
  */
 
-package za.co.absa.enceladus.common.plugin.menas
+package za.co.absa.enceladus.common.plugin.enceladus
 
-object MenasRunUrl {
+object EnceladusRunUrl {
 
-  def getMenasApiRunUrl(menasBaseUrl: String, datasetName: String, datasetVersion: Int, runNumber: Int): String =
-    s"$menasBaseUrl/api/runs/$datasetName/$datasetVersion/$runNumber"
+  def getApiRunUrl(baseUrl: String, datasetName: String, datasetVersion: Int, runNumber: Int): String =
+    s"$baseUrl/api/runs/$datasetName/$datasetVersion/$runNumber"
 
   def getMenasUiRunUrl(menasBaseUrl: String, datasetName: String, datasetVersion: Int, runNumber: Int): String =
```

Comment:
I am thinking about this method. Doesn't this force us to always have the same base URL for both API and UI? 
Otherwise, this is not true, right?

Reply:
Yes it does, I was thinking if fixing this should be in this PR or another one (as it requires addition of config field to `spark-jobs` properties, not just renaming). What do you think?